### PR TITLE
styling buttons slightly differently

### PR DESCRIPTION
**Pull request by 2is10**
@yipingliao

### CODE REVIEW

File: kifi-backend/modules/shoebox/angular/src/app.styl [hunk 1]
```diff
@@ -108,6 +108,33 @@ menu, ol, ul
...
+.kf-button-icon
+  display: inline-block
+  vertical-align: top
+  width: 18px
+  height: inherit
+  background-repeat: no-repeat
+  background-position: center center
+  background-size: contain
+  margin: 0 5px 0 -3px
+
```
Comment (2is10) [inline kifi-backend/modules/shoebox/angular/src/app.styl:137]: moved the styles from common.styl to app.styl so they aren’t repeated in our generated stylesheet (for each include)

Comment (yipingliao) [inline kifi-backend/modules/shoebox/angular/src/app.styl:137]: Nice!

File: kifi-backend/modules/shoebox/angular/src/libraries/manageLibrary.styl [hunk 2]
```diff
@@ -301,15 +301,7 @@ button.manage-lib-delete-button
     width: 100%
 
   .library-share-contact-button
-    // Override a bunch of global dialog button styles.
```
Comment (yipingliao) [inline kifi-backend/modules/shoebox/angular/src/libraries/manageLibrary.styl:304]: :)

File: kifi-backend/modules/shoebox/angular/src/app.styl (hunk 1 above)
Comment (2is10) [inline kifi-backend/modules/shoebox/angular/src/app.styl:137]: summary of styling changes:
- using `line-height` for vertical centering of text now
- no more absolute positioning of icons (padding no longer assumes there will be an icon)
- size of icon is now controlled only by its `kf-button-icon` width
- icon position can be adjusted using its margins or by translating or relatively positioning it
- no longer setting any margin on `kf-button`

**Comment by yipingliao** [comment]
:+1:

**Pull request merged**